src/main/java/com/gamebros/purepazaak/ai/SimpleAI.java

```diff
@@ -0,0 +1,135 @@
+package com.gamebros.purepazaak.ai;
+
+import java.util.ArrayList;
+import java.util.Optional;
+
+import com.gamebros.purepazaak.Board;
+import com.gamebros.purepazaak.Match;
+import com.gamebros.purepazaak.entity.Card;
+import com.gamebros.purepazaak.enumerable.PlayerEnum;
+
+public class SimpleAI {
+  protected Match match;
+
+  public SimpleAI(Match match) {
+    this.match = match;
+  }
+
+  public void update() {
+    // Stand if value equals 20
+    // Play a card from the side deck if it would make the value equal 20
+    // Play a card from the side deck to get as close to 20 as possible
+    // Stand as close to 20 as possible
+    // If the other play has stood, beat their score.
+
+    if (!this.isAiTurn()) {
+      return;
+    }
+
+    int currentValue = this.getCurrentValue();
+    int opponentValue = this.getOpponentValue();
+    ArrayList<Card> sideDeck = this.getSideDeck();
+
+    if (currentValue == 20) {
+      this.stand();
+
+      return;
+    }
+
+    if (this.opponentHasStood()) {
+      if (currentValue > opponentValue) {
```

Comment:
There's a bug here.  This condition should be `if (currentValue > opponentValue && currentValue <= 20)`.